vaadin-text-field-flow-parent/vaadin-text-field-flow/src/main/java/com/vaadin/flow/component/textfield/TextFieldVariant.java

```diff
@@ -15,14 +15,20 @@
  */
 package com.vaadin.flow.component.textfield;
 
+import com.vaadin.flow.component.ThemeVariant;
+
 /**
- * Set of theme variants applicable for {@code vaadin-text-field} component.
+ * Set of theme variants applicable for the {@link TextField} component, as well
+ * as other components based on it.
  */
-public enum TextFieldVariant {
-    LUMO_SMALL("small"), LUMO_ALIGN_CENTER("align-center"), LUMO_ALIGN_RIGHT(
-            "align-right"), LUMO_HELPER_ABOVE_FIELD(
-                    "helper-above-field"), MATERIAL_ALWAYS_FLOAT_LABEL(
-                            "always-float-label");
+public enum TextFieldVariant implements ThemeVariant {
+    //@formatter:off
+    LUMO_SMALL("small"),
+    LUMO_ALIGN_CENTER("align-center"),
+    LUMO_ALIGN_RIGHT("align-right"),
+    LUMO_HELPER_ABOVE_FIELD("helper-above-field"),
+    MATERIAL_ALWAYS_FLOAT_LABEL("always-float-label");
+    //@formatter:on
```

Comment:
Fixes the auto complete preview in IntelliJ showing whitespaces between the enum value, and the variant name string.